Makefile

```diff
@@ -26,25 +28,38 @@ install:
 	sudo install -Dm755 \
 		$(ROOT_DIR)/target/release/oxide \
 		$(ROOT_DIR)/target/release/oxide-bar \
+		$(ROOT_DIR)/target/release/oxide-config \
 		$(ROOT_DIR)/target/release/oxide-msg \
 		-t $(TARGET_DIR)
 	sudo cp -t $(CONFIG_DIR)/oxide/ \
 		$(ROOT_DIR)/resources/config.yml \
 		$(ROOT_DIR)/bar_config.yml
 	sudo install -Dm644 $(ROOT_DIR)/resources/oxide.desktop $(SHARE_DIR)/xsessions/oxide.desktop
-	sudo cp $(ROOT_DIR)/man/oxide-msg.1 $(ROOT_DIR)/man/oxide.1 $(SHARE_DIR)/man/man1/
+
+	$(call gen_manpages)
+	sudo cp $(ROOT_DIR)/man/oxide.1 \
+		$(ROOT_DIR)/man/oxide-bar.1 \
+		$(ROOT_DIR)/man/oxide-config.1 \
+		$(ROOT_DIR)/man/oxide-msg.1 \
+		$(SHARE_DIR)/man/man1/
+
```

Comment:
If you create a folder where you generate the manpages to, you could simply copy the folder ?:)

Reply:
@FelixSchladt ?

Reply:
still manual adding is required in the uninstall phase, i would just leave it as is